apps/charterafrica/src/components/OpportunityCardList/OpportunityCardList.js

```diff
@@ -0,0 +1,115 @@
+import Link from "@/commons-ui/next/Link";
+import { Typography, Box, Grid } from "@mui/material";
+import { useTheme } from "@mui/material/styles";
+import useMediaQuery from "@mui/material/useMediaQuery";
+import React from "react";
+
+import OpportunityCard from "../OpportunityCard";
+
+import { neutral } from "@/charterafrica/colors";
+
+const OpportunityCardList = React.forwardRef(function OpportunityCardList(
+  props,
+  ref
+) {
+  const { title, grants } = props;
+  const theme = useTheme();
+  const isMobile = useMediaQuery(theme.breakpoints.down("sm"));
+  const isSmallDesktop = useMediaQuery(theme.breakpoints.only("md"));
```

Comment:
This technique was useful in MUI v4. In MUI v5, `sx` prop on a component handle most cases.

... also remember this technique is done **after** JS has been fully downloaded and executed on the client where `sx` is pure CSS.